### DISCUSSION TITLE
Cluster doesn't start on bare metal UPI

### DISCUSSION THREAD

**Discussion by fs30000** [okd-project/okd#1236] [General]
Hello guys,

Would appreciate so very much some light on this. I am have been pulling my hair for more than one week already :) In fact almost 2 weeks now.

I am following openshift's/redhat's doc for this bare metal user provisioned instalation. Among other online howto's.

Details:

- Using KVM/libvirt on one physical machine.
- 6 VMs created, bootstrap, 3 control nodes and 2 computes nodes. Enough RAM for all.
- VMs are installed within 24h of the ignition files creation (otherwise it would throw certificate errors)
- The VMs are installed via PXE with the specific ignition files. First i install the bootstrap, then the 3 control nodes and then the 2 workers.

- The basic networking services are installed and running correctly ( i hope lol). DHCP, DNS (with PTR), PXE, HTTP, etc.
- Haproxy config is right. Pointing to the right servers/machines.
- Certs are approved.
- "oc get nodes" only show the masters. They show as Ready, but haproxy's /stats shows otherwise for the 6443 port. Sometimes, after reinstalling all VMs, the workers also show up. Sometimes they even show up as Ready too, but "openshift-install --dir /root/os wait-for bootstrap-complete   --log-level=debug" show a fail.

- Latest openshift-install and oc.  openshift-install 4.10.15
- Latest RHCOS, initrmfs, roofs and kernel. 410.84.202201251210-0
- pullsecret is one week old. Does that matter?

[.openshift_install.log](https://github.com/openshift/okd/files/8788761/default.openshift_install.log)
[install-config-yaml.txt](https://github.com/openshift/okd/files/8788762/install-config-yaml.txt)

**Comment by fs30000** [comment]
Ok, i increased the CPUs for each VM from 2 to 4 and now the cluster started! About time :) I even tested OKD as well. Same good result.

Why does it use so much cpu even when iddle?

**Comment by vrutkovs** [comment]
>Latest openshift-install and oc. openshift-install 4.10.15
>Latest RHCOS, initrmfs, roofs and kernel. 410.84.202201251210-0

You're installing OCP, not OKD

**Reply by fs30000** [reply]
So, this site is only for OKD, right? My apologies, i was desperate. I thought it was  almost the same thing.

**Reply by vrutkovs** [reply]
Yeah, this repo is for OKD specific bugs. Contact Red Hat support for OCP issues
